### PR TITLE
added missing parameter descriptions

### PR DESCRIPTION
**Pull request by fiedsch**
found some descriptions where not all parameters the example showed were described

### CODE REVIEW

**Comment by fiedsch** [comment]
Just found out that this PR also includes the change from #342 -- sorry I messed the creation of a new branch.

File: api/extensions/hooks/getImage.md [hunk 1]
```diff
@@ -20,21 +20,29 @@ cache name and the file object as arguments and expects a path as return value.
...
 
-6. *string* `$target`
+7. *string* `$target`
 
     The target location where the image should be stored. This will be `null` in
     most cases, if not you should ignore `$strCacheName`.
 
+8. *object* `$objElement`
+ 
+    The Image content element.
```
Comment (aschempp) [inline api/extensions/hooks/getImage.md]: That's not correct. It's not the content element, it's the instance of `Contao\Image` class.

File: api/extensions/hooks/addComment.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ the record and the data array as arguments and does not expect a return value.
 2. *array* `$arrSet`
 
     Data of the new comment record (not including the ID).
+    
+3. *object* `$objComments`
+
+    The comments module that was used to add the comment.
```
Comment (aschempp) [inline api/extensions/hooks/addComment.md]: Same thing here. It's the instance of the comments class, but not a module.

File: api/extensions/hooks/activateAccount.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ It passes the user object as argument and does not expect a return value.
 1. *Database_Result* `$objUser`
 
     A database result set of the activated user account.
+    
+2. *object* `$objRegistration`
```
Comment (aschempp) [inline api/extensions/hooks/activateAccount.md]: I would call this `$objModule` to be more clear.

**Comment by fiedsch** [comment]
Thank you for your input! 
I updated the PR to include the suggested corrections.

**Comment by aschempp** [comment]
Thanks!

**Pull request merged**